base/src/main/java/com/smartdevicelink/streaming/video/VideoStreamingParameters.java

```diff
@@ -186,12 +194,15 @@ public void update(VideoStreamingCapability capability, String vehicleMake) {
             int capableBitrateInKb = Math.min(Integer.MAX_VALUE / 1000, capability.getMaxBitrate());
             this.bitrate = Math.min(this.bitrate, capableBitrateInKb * 1000);
         } // NOTE: the unit of maxBitrate in getSystemCapability is kbps.
-        double scale = DEFAULT_SCALE;
-        // For resolution and scale, the capability values should be taken rather than parameters specified by developers.
         if (capability.getScale() != null) {
             scale = capability.getScale();
         }
+        if (capability.getDiagonalScreenSize() != null) { preferredDiagonal = capability.getDiagonalScreenSize(); }
```

Comment:
```suggestion
        if (capability.getDiagonalScreenSize() != null) {
            preferredDiagonal = capability.getDiagonalScreenSize(); 
        }
```

To maintain consistent formatting